packages/synapse-react-client/src/components/AccessRequirementList/AccessRequirementList.tsx

```diff
@@ -243,64 +267,53 @@ export default function AccessRequirementList(
           File(s)
         </>
       )
-    return (
-      <>
-        <IconSvg icon="file" sx={{ width: '30px' }} />{' '}
-        <Link
-          href={`${PRODUCTION_ENDPOINT_CONFIG.PORTAL}#!Synapse:${entityId}`}
-        >
-          {entityInformation[0]?.name}
-        </Link>
-      </>
-    )
-  }, [entityId, entityInformation, numberOfFilesAffected, requestObjectName])
+    if (subjectType === RestrictableObjectType.ENTITY) {
+      return <EntityLink entity={subjectId} />
+    } else if (subjectType === RestrictableObjectType.TEAM) {
+      return <UserOrTeamBadge principalId={subjectId} />
+    } else {
+      console.warn(
+        `Unhandled case for AccessRequirementList requestDetails: ${subjectType}: ${subjectId}`,
+      )
+      return <></>
+    }
+  }, [numberOfFilesAffected, requestObjectName, subjectId, subjectType])
 
   const content = (
     <>
```

Comment:
Remove `DialogTitle` and `DialogContent` from `content`. These will now be handled by our `DialogBase` component, and the wrapper components will not appear when `renderAsModal` is false.